app/controllers/container_build_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ContainerBuildController < ApplicationController
```

Comment:
I noticed most of our container controllers share some basic methods, in this case ```show_list ``` and ```display_name```.
I think I'll create a common container controller, to hold these methods. @simon3z do you object?